Author: Pino Toscano <pino@debian.org>
Description: Make QtWebEngine an optional requirement
 This makes it possible to build kaccount-provides, albeith without the
 nextcloud plugin, also on architectures that do not have QtWebEngine.
Forwarded: no
Last-Update: 2020-08-14

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -16,7 +16,8 @@ set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_S
 find_package(Intltool REQUIRED)
 find_package(KAccounts REQUIRED)
 
-find_package(Qt5 ${QT_REQUIRED_VERSION} CONFIG REQUIRED Core Qml WebEngineWidgets)
+find_package(Qt5 ${QT_REQUIRED_VERSION} CONFIG REQUIRED Core Qml)
+find_package(Qt5WebEngineWidgets ${QT_REQUIRED_VERSION} CONFIG)
 find_package(KF5 ${KF5_MIN_VERSION} REQUIRED KIO I18n Declarative Package)
 
 include(KDEInstallDirs)
--- a/plugins/CMakeLists.txt
+++ b/plugins/CMakeLists.txt
@@ -1,2 +1,4 @@
 add_subdirectory(owncloud-ui)
-add_subdirectory(nextcloud-ui)
\ No newline at end of file
+if (Qt5WebEngineWidgets_FOUND)
+  add_subdirectory(nextcloud-ui)
+endif()
